modules/ROOT/pages/zephyr-sample.adoc

```diff
@@ -11,6 +11,9 @@ In this section, we assume you have already set up your development environment
 === Blinky sample
 The first sample is *Blinky*. This application blinks an LED on your Development Board.
 
+CAUTION: For the *Blinky* sample, make sure that SW1 Bit 8 (LED control) is turned ON.
```

Comment:
We talked about how we should explain the position / pole / bit of the dip switches. and we decided to go with the notaion of `SW1[8]`.